tests/test_views.py

```diff
@@ -4970,7 +4972,8 @@ def test_can_reset_password_if_form_success(self, mock_service_validate):
         )
         ticket = fake.md5()
         service_url = 'http://accounts.osf.io/?ticket=' + ticket
-        cas.make_response_from_ticket(ticket, service_url)
+        with run_celery_tasks():
+            cas.make_response_from_ticket(ticket, service_url)
```

Comment:
I was thinking at first that `verification_key` must have been used in many other places where our unit tests don't cover. But then I realized that the change only affects the verification key clearing process, which only happens during `. make_response_from_ticket()`.